packages/react-storage/src/components/StorageBrowser/controls/__tests__/ActionCancelControl.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ActionCancelControl } from '../ActionCancelControl';
+import * as useActionCancelModule from '../hooks/useActionCancel';
+
+describe('ActionCancelControl', () => {
+  const useActionCancelSpy = jest.spyOn(
+    useActionCancelModule,
+    'useActionCancel'
+  );
+
+  afterEach(() => {
```

Comment:
nit: afterEach seems odd compared to using beforeEach for this. I don't know if there's really a difference here, so probably just personal pref

Reply:
can go with beforeEach